main.tf

```diff
@@ -74,6 +74,10 @@ resource "aws_iam_role_policy" "datafy" {
           "ec2:CreateVolume",
           "ec2:CreateSnapshot",
           "ec2:CreateSnapshots",
+          "ebs:StartSnapshot",
+          "ebs:PutSnapshotBlock",
+          "ebs:CompleteSnapshot",
+          "ebs:ListSnapshotBlocks",
```

Comment:
I think it needs to go to the above block

Reply:
This is only used as part of the snapshot flow, not in read only mode.